tests/queries/0_stateless/00002_log_and_exception_messages_formatting.sql

```diff
@@ -1,4 +1,4 @@
--- Tags: no-parallel, no-fasttest, no-ubsan, no-batch
```

Comment:
Could you please write a small comment similar to `no-parallel` about why we don't want to run this test in flaky test check?